modules/sdk-lib-mpc/src/tss/ecdsa/index.ts

```diff
@@ -1,3 +1,5 @@
 export * as EcdsaTypes from './types';
 export * as EcdsaRangeProof from './rangeproof';
 export * as EcdsaPallierProof from './pallierproof';
+
+export const minModulusBitLength = 3072;
```

Comment:
nit: will benefit from changing all occurrences of 3072 to minModulusBitLength, some places are still not changed.